RestAPI.cs

```diff
@@ -43,12 +43,12 @@ public class RestAPI
             if (string.IsNullOrEmpty(url))
                 throw new Exception("Please use a valid WooCommerce Restful API url.");
 
-            url = url.Trim().ToLower().TrimEnd('/');

```

Comment:
`url = url.Trim().ToLower().TrimEnd('/');`

This line already has ToLower(), what happened here?